storage/blockchain/src/service/mod.rs

```diff
@@ -36,9 +36,10 @@
 //! - The last [`BCReadHandle`] is dropped => reader thread-pool exits
 //! - The last [`BCWriteHandle`] is dropped => writer thread exits
 //!
-//! Upon dropping the [`cuprate_database::Env`]:
+//! TODO: update this when `ConcreteEnv` is removed
+//! Upon dropping the [`cuprate_database::ConcreteEnv`]:
```

Comment:
Not formatted properly for `cargo doc`.
```suggestion
//! TODO: update this when `ConcreteEnv` is removed.
//!
//! Upon dropping the [`cuprate_database::ConcreteEnv`]:
```